great_expectations/datasource/data_connector/data_connector.py

```diff
@@ -116,9 +123,14 @@ def build_batch_spec(self, batch_definition: BatchDefinition) -> BatchSpec:
                 batch_definition=batch_definition
             )
         )
-        batch_spec_passthrough: dict = batch_definition.batch_spec_passthrough
-        if isinstance(batch_spec_passthrough, dict):
-            batch_spec_params.update(batch_spec_passthrough)
+        # batch_spec_passthrough via Data Connector config
+        batch_spec_passthrough: dict = deepcopy(self.batch_spec_passthrough)
+
+        # batch_spec_passthrough from batch_definition supercedes batch_spec_passthrough from Data Connector config
+        if isinstance(batch_definition.batch_spec_passthrough, dict):
+            batch_spec_passthrough.update(batch_definition.batch_spec_passthrough)
+
```

Comment:
thank you for these inline comments :)

Reply:
Is it possible to specify batch_spec_passthrough for a given data asset (as opposed to all assets within a configuration) when creating the datasources configuration in "great_expectations.yml". 

When trying to add a batch_spec_passthrough to an asset within a connector, it gives an error saying "Asset" is not subscriptible.